src/models/ILine.ts

```diff
@@ -1,8 +1,15 @@
 import TransitType from '~/enums/transitType';
 import { ILineRoute } from '.';
 
-export default interface ILine {
-    transitType: TransitType;
+interface ILinePrimaryKey {
```

Comment:
export interface ILinePrimaryKey

is how we have exported enums / interfaces before, if you look for example at networkStore